CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased 
+
+* Dependency updates. Upgraded to `sw-precache@5.1.1` minimum to prevent error about corrupted data on firefox 52 and Chrome 60 (witch match the spec)
```

Comment:
I believe that this should be Chrome 59. Probably want to capitalize Firefox. Did you mean 'which match the spec'?